packages/backend/src/handlers/actions/migrateSourceCredAccounts/handler.ts

```diff
@@ -71,13 +72,32 @@ const parseAlias = (alias: SCAlias) => {
   }
 };
 
+const storage = new sc.ledger.storage.GithubStorage({
+  apiToken: CONFIG.githubApiToken,
+  repo: 'MetaFam/XP',
+  branch: 'master',
+});
+
+const manager: LedgerManager = new sc.ledger.manager.LedgerManager({
+  storage,
+});
+
+const ledgerLoadedPromise = manager.reloadLedger();
+
 export const migrateSourceCredAccounts = async (
   _: Request,
   res: Response,
 ): Promise<void> => {
-  const accountsData: SCAccountsData = await (
-    await fetch(ACCOUNTS_FILE)
-  ).json();
+  const ledgerRes: ReloadResult = await ledgerLoadedPromise;
```

Comment:
```suggestion
  const ledgerRes: ReloadResult = await ledgerLoadedPromise();
```

Reply:
`ledgerLoadedPromise` is of type `Promise<ReloadResult>`.  It is different syntax than usual, but correct.

Usually one would write something like `const ledgerLoadedResult = await manager.reloadLedger()`; Here I'm just assigning the promise to a variable, and awaiting that. 

The benefit here is that `reloadLedger` is called when the backend boots up. We await it to ensure that it is complete before the handler proceeds with execution.

Reply:
Gotcha. Very cool. Thanks for adding a trick to my quiver.